src/ScottPlot4/ScottPlot.Avalonia/README.md

```diff
@@ -14,7 +14,6 @@ Add the following to your start-up sequence:
 double[] dataX = new double[] { 1, 2, 3, 4, 5 };
 double[] dataY = new double[] { 1, 4, 9, 16, 25 };
 
-AvaPlot avaPlot1 = this.Find<AvaPlot>("AvaPlot1");
 avaPlot1.Plot.AddScatter(dataX, dataY);
```

Comment:
Should be `AvaPlot1` since the variable is gone now.

Reply:
Good catch, will be fixed.